src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -980,7 +980,9 @@ def main(argv):
 
   # Make a decision on whether merge step is needed at all. If there are no
   # new units added by libFuzzer run, then no need to do merge at all.
-  new_units_added = parsed_stats.get('new_units_added', 0)
+  new_units_added = (
+      shell.get_directory_file_count(new_testcases_directory) or
+      parsed_stats.get('new_units_added'))  # redundant, but needed for tests.
```

Comment:
Can you change the comment to a TODO to mock `shell.get_directory_file_count`. I think this is fine as a temporary solution but seems weird to have long term unless absolutely necessary.

Reply:
+1. mocking is definitely better here.

Reply:
fixed.
minijail test was broken since self.command[0] is not _fuzzer there but minijail bin path.
also test_oom test ends up showing case where merge units=0, so we have a way to emulate that using None in mock_popen, so used that.